apps/keyboard/js/layouts/uk.js

```diff
@@ -11,6 +11,7 @@ Keyboards.uk = {
     alt: {
       а: '@',
       г: 'ґ',
+      і: 'ї',
       '.': ',?!;:…’'
     },
     width: 12,
```

Comment:
All of the rows are of 11 keys... I am not entirely sure any effect if you keep the `width` 12. If this is fine we could keep it as-is.